plugins/formal-verification/formver.core/src/org/jetbrains/kotlin/formver/embeddings/NameEmbeddings.kt

```diff
@@ -31,19 +33,51 @@ data class ClassName(val packageName: FqName, val className: Name) : MangledName
         get() = "pkg_${packageName.asString()}\$class_${className.asString()}"
 }
 
-data class ClassMemberName(val className: ClassName, val name: Name) : MangledName {
+open class ClassMemberName(val className: ClassName, val name: Name) : MangledName {
     override val mangled: String
         get() = "${className.mangled}\$member_${name.asString()}"
 }
 
-/** Global name not associated with a class.
+class ClassMemberAccessor(className: ClassName, name: Name, val accessor: AccessorType) : ClassMemberName(className, name) {
+    enum class AccessorType {
+        GETTER, SETTER;
+
+        override fun toString(): String = when (this) {
+            GETTER -> "getter"
+            SETTER -> "setter"
+        }
+    }
+
+    override val mangled: String
+        get() = "${super.mangled}_$accessor"
```

Comment:
I think this could cause name conflicts.

Reply:
Yes